### PR TITLE
[CP] Katello 3.12.1 Cherrypicks from Toolbelt

### PR DESCRIPTION
**Pull request by chris1984**

### CODE REVIEW

**Comment by chris1984** [comment]
@jturel it looks like tests passed except angular-js so the vcr stuff worked

**Comment by jturel** [comment]
@chris1984 wanna take a shot at finding the commit that fixes those lint errors?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,30 @@
+# 3.12.1 Iron Stout (2019-07-09)
+
+## Features
```
Comment (jturel) [inline CHANGELOG.md]: Let's remove the features heading since it's empty

**Comment by chris1984** [comment]
@jturel updated!

**Comment by jturel** [comment]
Looks good! I'll just wait for confirmation from you before merging and releasing the gem - just in case. Lemme know

**Comment by chris1984** [comment]
@jturel I ran the cherry-picks again and dont see anything new, I think we are good to merge and release the gem.

**Comment by jturel** [comment]
@chris1984 

3.12.1 gem: https://rubygems.org/gems/katello/versions/3.12.1
3.12.1 tag: https://github.com/Katello/katello/tree/3.12.1

**Pull request merged**